libraries/ae-inventory-processor/src/main/java/org/metaeffekt/core/inventory/processor/patterns/contributors/WebModuleComponentPatternContributor.java

```diff
@@ -44,9 +44,10 @@ public class WebModuleComponentPatternContributor extends ComponentPatternContri
         add("/.package-lock.json");
```

Comment:
We should consider to also allow .package-lock.json files in the root of our scan directly. We should discuss this. Effectively it may mean removing the trailing slashes here (and probably in other contributors).